lib/Core/Searcher.cpp

```diff
@@ -287,12 +295,70 @@ void
 RandomPathSearcher::update(ExecutionState *current,
                            const std::vector<ExecutionState *> &addedStates,
                            const std::vector<ExecutionState *> &removedStates) {
-}
+  size = size + addedStates.size() - removedStates.size();
+  if (current != nullptr) {
```

Comment:
Instead of checking `current` you could just inherit tags in `attach()`. The last argument is always the parent/current state - but we would need to enforce/document this.